save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/projects/ProjectDemoMenu.kt

```diff
@@ -176,11 +180,15 @@ val projectDemoMenu: FC<ProjectDemoMenuProps> = FC { props ->
                 div {
                     className = ClassName("flex-wrap d-flex justify-content-around")
                     when (demoStatus) {
-                        DemoStatus.NOT_CREATED -> buttonBuilder("Create") {
-                            sendDemoCreationRequest()
+                        DemoStatus.NOT_CREATED -> if (getHighestRole(props.organizationRole, props.projectRole).isHigherOrEqualThan(Role.OWNER)) {
+                            buttonBuilder("Create") {
+                                sendDemoCreationRequest()
+                            }
```

Comment:
This is not how it should done: take a look at what `disabled` (or `isDisabled`, don't remember) does